script/test/stress.js

```diff
@@ -0,0 +1,36 @@
+import http from 'k6/http';
+import { check, group, sleep, fail } from 'k6';
+
+export let options = {
+    stages: [
+        { duration: '5m', target: 50 },
+        { duration: '5m', target: 50 },
+        { duration: '5m', target: 100 },
+        { duration: '5m', target: 100 },
```

Comment:
```suggestion
        { duration: '20s', target: 50 },
        { duration: '5m', target: 50 },
        { duration: '20s', target: 100 },
        { duration: '5m', target: 100 },
```

ramp-up 구간은 좀 더 짧게 가져가도 괜찮을듯해요